sample/index.html

```diff
@@ -33,6 +33,13 @@
         <meta name="msapplication-tap-highlight" content="no">
         <meta name="viewport" content="user-scalable=no, initial-scale=1, maximum-scale=1, minimum-scale=1, width=device-width">
         <link rel="stylesheet" type="text/css" href="css/index.css">
+        <style type="text/css" media="screen">
+            button {
+                height: 50px;
+                width: 80%;
+                margin: 5px;
+            }
+        </style>
```

Comment:
Can’t you add it to the index.css file rather that adding inline styles

Reply:
`index.css` generated by cordova tool (during boilerplate project creation) so we cannot modify it, technically we can have own index.css but in this case we will need to support it